di/src/main/java/com/interface21/context/support/AnnotatedBeanDefinition.java

```diff
@@ -0,0 +1,23 @@
+package com.interface21.context.support;
+
+import com.interface21.beans.factory.support.GenericBeanDefinition;
+import com.interface21.beans.factory.support.InjectMode;
+import java.lang.reflect.Method;
+
+public class AnnotatedBeanDefinition extends GenericBeanDefinition {
+
+    private final Method method;
+    public AnnotatedBeanDefinition(Class<?> bean, Method method) {
```

Comment:
```suggestion
    private final Method method;
    
    public AnnotatedBeanDefinition(Class<?> bean, Method method) {
```

놓치신 것 같아요~
상태값과 초기화부분의 영역은 자바 스타일 가이드에 따라 구분지어주면 좋습니다 :)

Reply:
수정하도록 하겠습니다!